README.md

```diff
@@ -1,3 +1,21 @@
+# Deprecation Notice
+
+This buildpack is *deprecated* as it's no longer necessary.
```

Comment:
s/as it's no longer necessary/has been superseded by functionality built into Heroku/ ?